### DISCUSSION TITLE
PHP Ini Analyzer for CLI?

### DISCUSSION THREAD

**Discussion by gauravmak** [enlightn/enlightn#82] [Q&A] [answered]
Why is it required to change the settings for the CLI as well on [this page](https://www.laravel-enlightn.com/docs/security/php-ini-analyzer.html)?

Someone can access PHP via CLI only after logging into the server, right?

**Comment by paras-malhotra** [comment, marked as answer]
@gauravmak could also access it as part of a cron job that relies on user input
